consensus/polybft/transport.go

```diff
@@ -0,0 +1,117 @@
+package polybft
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/0xPolygon/go-ibft/messages/proto"
+	pbftproto "github.com/0xPolygon/polygon-edge/consensus/polybft/proto"
+	"github.com/0xPolygon/polygon-edge/network"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+// Bridge transport is an abstraction of network layer for a bridge
+type BridgeTransport interface {
+	Multicast(msg interface{})
+}
+
+type runtimeTransportWrapper struct {
+	bridgeTopic *network.Topic
+	logger      hclog.Logger
+}
+
+var _ BridgeTransport = (*runtimeTransportWrapper)(nil)
+
+// Multicast publishes any message as pbftproto.TransportMessage
+func (g *runtimeTransportWrapper) Multicast(msg interface{}) {
+	data, err := json.Marshal(msg)
+	if err != nil {
+		g.logger.Warn("failed to marshal bridge message", "err", err)
+
+		return
+	}
+
+	err = g.bridgeTopic.Publish(&pbftproto.TransportMessage{Data: data})
+	if err != nil {
+		g.logger.Warn("failed to gossip bridge message", "err", err)
+	}
+}
+
+// subscribeToBridgeTopic subscribes for bridge topic
+func (cr *consensusRuntime) subscribeToBridgeTopic(topic *network.Topic) error {
+	return topic.Subscribe(func(obj interface{}, _ peer.ID) {
+		msg, ok := obj.(*pbftproto.TransportMessage)
+		if !ok {
+			cr.logger.Warn("failed to deliver message", "err", "invalid msg")
```

Comment:
do we need to log the object itself? also is it a warning or an error?

Reply:
the same below.

Reply:
Fixed [c14c710](https://github.com/0xPolygon/polygon-edge/pull/843/commits/c14c7100feaa8b6d220905a0b02c03632ca046ca)